### PR TITLE
Update RTA Lambda to .NET Core 3.1

### PR DESCRIPTION
**Pull request by DBurkhead**
*Description of changes:*
Update projects to target .NET Core 3.1.
Update NuGet package dependencies.
Update Lambda runtime in template to 3.1.
Rebuild binaries and update package .zip file.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .taskcat.yml [hunk 1]
```diff
@@ -19,8 +16,5 @@ tests:
       QSS3BucketRegion: $[taskcat_current_region]
     regions:
     - us-east-1
-    - us-west-2
-    - eu-central-1
-    - ap-southeast-2
-    s3_bucket: ''
+    s3_bucket: 'quickstart-reference-dburkhead'
```
Comment (andrew-glenn) [inline .taskcat.yml:19]: Same here

File: .taskcat.yml [hunk 2]
```diff
@@ -4,10 +4,7 @@ project:
   package_lambda: false
   regions:
   - us-east-1
-  - us-west-2
-  - eu-central-1
-  - ap-southeast-2
-  s3_bucket: ''
+  s3_bucket: 'quickstart-reference-dburkhead'
```
Comment (andrew-glenn) [inline .taskcat.yml:7]: Please replace this with `$[taskcat_autobucket]`; Our tests will run and attempt to upload to your bucket, and will fail. `$[taskcat_autobucket]` is a psuedo-parameter placeholder for an auto-generated bucket name.

**Review by andrew-glenn — CHANGES_REQUESTED** [review]
Minor change requested. The rule failing I'm going to dig into/fix and then get this passing.

File: .taskcat.yml (hunk 2 above)
Comment (DBurkhead) [reply .taskcat.yml:7]: @andrew-glenn Added another commit to switch to $[taskcat_autobucket]. Let me know if you see any other changes required.

**Comment by andrew-glenn** [comment]
Rule exception fixed in https://github.com/aws-quickstart/qs-cfn-lint-rules/pull/17

**Review by andrew-glenn — APPROVED** [review]

**Pull request merged**